packages/manager/modules/pci/src/components/pci-order-command-terraform/pci-order-command-terraform.controller.js

```diff
@@ -0,0 +1,43 @@
+export default class PciOrderCommandTerraformCtrl {
+  $onInit() {
+    this.terraformDisplay = this.formatTerraformNode(this.terraformData);
+  }
+
+  formatTerraformNode(node, identationCount = 0) {
```

Comment:
```suggestion
  formatTerraformNode(node, identationCount = 4) {
```

Reply:
To be treated with https://jira.ovhcloud.tools/browse/DATATR-236